### ISSUE TITLE
The Pinot benchmark does not have indices.

### ISSUE THREAD

**Issue by didip** [ClickHouse/ClickBench#37]
I am not a user of Pinot so unfortunately I cannot help in this case.

But it seems like it should have indices.

**Comment by alexey-milovidov** [comment]
What indices to add?

**Comment by alexey-milovidov** [comment]
If you can take these changes and make them ready for a pull request, it will be helpful.

**Comment by MacNale** [comment]
@alexey-milovidov  https://docs.pinot.apache.org/basics/indexing

Can you please bump up the pinot version to 0.12.1?

**Comment by alexey-milovidov** [comment]
Yes, ClickBench can install any version; please edit this file: https://github.com/ClickHouse/ClickBench/blob/main/pinot/benchmark.sh

Then you can send the results, or we can run it.

**Comment by MacNale** [comment]
okay.. Let me take a stab it. I am not a pinot expert. I am trying to evaluate Pinot vs clickhouse for my use case with my own data.  This benchmark is a good start.   

As per this blog 'https://startree.ai/blog/a-tale-of-three-real-time-olap-databases', pinot performs better than clickhouse. Nevertheless, let me read about pinot indexes before I come up with suggestions.

**Comment by alexey-milovidov** [comment]
We stumbled upon this article and found many factual errors and misrepresentations.

We did not find any reproducible scenario where Pinot can be faster than ClickHouse.
Moreover, Pinot is much inferior in usability - you will find obstacles starting with installation and data loading.

**Comment by rschu1ze** [comment]
@wisercoder Fair enough. Unfortunately, none of the guys at ClickHouse is an expert in Pinot tuning. Feel free to send a PR that adds indices to Pinot's ClickBench DDL. I am happy to try it out and update benchmark results.
